template/README.md

```diff
@@ -96,6 +96,15 @@ $ npm run serve
 
 Open [localhost:8000](http://localhost:8000/)
 
+### Writing SCSS
+Auro has implemented [CSScomb](https://www.npmjs.com/package/csscomb) for style formatting enforcement in addition to the [linting requirements](https://auro.alaskaair.com/webcorestylesheets/linter).
+
+The build process will auto-execute the CSScomb formatter as part of the pre-commit workflow prior to executing the automated test scripts. If your component includes more than a trivial amount of custom CSS you may find it helpful to run the CSSComb formatter during your regular development. You can run `npm run cssComb` to execute the process manually.
+
+Additionally, you may choose to install a CSScomb extension for your IDE of choice. These extensions allow for simple format-on-save and keybind trigger of CSScomb execution. These extensions will automatically use the config file located at the root of the project template and format your styles in compliance with Auro guidelines. Refer to your chosen IDE's documentation for how to search for and install extensions.
```

Comment:
I am having concerns about this as a utility. The Github repo seems to have been abandoned and the repo for the VSCode plugin is archived. 

I have concerns about adopting this and then having to remove it in a year.

Reply:
The repo is not abandoned - https://github.com/csscomb/csscomb.js - and there are 26,000 weekly installs of the npm CSScomb package.

Reply:
The thing that concerns me is that the last updates to the repo were 2 years ago, and that was to bump to 4.3.

I am not disregarding its popularity. I am concerned about its support.

Reply:
_**TL;DR** - I don't think we necessarily should be enforcing this with the commits - but I do think it's beneficial to contributors and maintainers to provide the tooling in place to to ease their efforts in achieving our stated requirements and ideal formatting of code._

I don't have any particular attachment to this being an enforced part of the build which is why I didn't advocate for this change to begin with. We don't particularly have to enforce that like we want to with our lint rules. It doesn't ultimately matter during execution if `0px` or `0` was used for a property and it doesn't matter if if the top property is assigned before the position property for a given selector - the browser will still figure it out. I love to have it consistent and I think there is "better (IMO)" way to format the code but I also don't think it's worth opening up to a possible debate with every individual about what the right format to use is or how to go about getting there as this is often an arduous debate. Similarly, I got bored with the tabs vs. spaces and hard G vs soft G in "GIF" almost as soon as those heated debates started. 

We should at a minimum have a style guide to promote the preferred formatting and leave it up to the contributor to decide for themselves how they achieve that goal. If someone wants to do it manually, good for them, that's great. If someone wants to use a utility to simplify the process, great. I happen to use CSScomb to achieve the desired result because it's simple to use and a big time saver. If CSScomb were to stop working at some point I would certainly be disappointed. But it wouldn't change the reason I used it and I would still be striving to format my styles in the same manner one way or another.

By establishing a style guide for Auro - providing a config file for formatting tools that a developer may use is at least a polite courtesy. To me this would mean adding the NPM package, the config file and a script for use with `npm run ...` but not enforce the use of it.